### PR TITLE
Resource aws_iam_root_user; Property:access_key_count correction

### PR DESCRIPTION
**Pull request by rx294**
Resource aws_iam_root_user exposes ` summary_account['AccountAccessKeysPresent']` as `access_key_count` which is not a count but a boolean value that checks if AccessKeys are present

```
Current Code:
  def access_key_count
    summary_account['AccountAccessKeysPresent']
  end

Changed to Code:
  def has_access_key?
    summary_account['AccountAccessKeysPresent'] == 1
  end
```

### CODE REVIEW

**Comment by adamleff** [comment]
Please use more descriptive PR titles and always include a description. A user should not have to refer to another issue in order to determine the scope and intention of a pull request. Thank you!

**Comment by rx294** [comment]
Thank you @adamleff I have made the changes.

**Review by clintoncwolfe — CHANGES_REQUESTED** [review]
Just a couple of docs changes - looks good.  I've run unit and integration tests, both pass fine.

File: docs/resources/aws_iam_root_user.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ The following examples show how to use this InSpec audit resource.
 ### Test that the AWS root account has only one access key
```
[the remark this answers is not in the document]
Comment (rx294) [reply docs/resources/aws_iam_root_user.md]: Ah..sorry meant to say at-least...

File: docs/resources/aws_iam_root_user.md [hunk 2]
```diff
@@ -49,3 +49,9 @@ This InSpec audit resource has the following special matchers. For a full list o
 The `have_mfa_enabled` matcher tests if the AWS root user has Multi-Factor Authentication enabled, requiring them to enter a secondary code when they login to the web console.
 
     it { should have_mfa_enabled }
+
+### have_access_key
+
+The `have_access_keys` matcher tests if the AWS root user has at least one access key.
```
[the remark this answers is not in the document]
Comment (rx294) [reply docs/resources/aws_iam_root_user.md]: another oops!

**Comment by rx294** [comment]
Thanks for the review @clintoncwolfe ...I have made the corrections...apparently I suck at documentation 😆

**Review by clintoncwolfe — APPROVED** [review]

**Comment by clintoncwolfe** [comment]
Thanks again - great contribution!

**Comment by rx294** [comment]
Thank you for the approval!

**Pull request merged**